README.md

```diff
@@ -84,4 +84,4 @@ The _"non-endorsed"_ status means that the plugin is not endorsed by the origina
 | [**video_player_tizen**](packages/video_player) | 4.0 | ✔️ | ✔️ | ⚠️ | ❌ | Functional limitations,<br>TV emulator issue |
 | [**wakelock_tizen**](packages/wakelock) | 4.0 | ✔️ | ✔️ | ❌ | ❌ | Cannot override system settings |
 | [**wearable_rotary**](packages/wearable_rotary) | 4.0 | ✔️ | ✔️ | ❌ | ❌ | Not applicable for TV |
-| [**webview_flutter_tizen**](packages/webview_flutter) | 5.5 | ✔️ | ✔️ | ✔️ | ✔️ | Not for production use |
+| [**webview_flutter_tizen**](packages/webview_flutter) | 5.5 | ❌ | ❌ | ✔️ | ❌ | Only for TV Devices |
```

Comment:
Incompatible with emulators?

Also what's the rationale for the minimum API version (5.5)?

Reply:
> Incompatible with emulators?

Yes, As far as I know, the behavior of the internal API (`ewk_view_offscreen_rendering_enabled_set`) used for the implementation is not guaranteed in the emulator and it doesn't actually work.
cc. @bwikbs 

> Also what's the rationale for the minimum API version (5.5)?

This is the highest Tizen version required among the internal APIs used in this plugin. 
Um... But now, this plugin only  supports TV device and since flutter-tizen support TV with Tizen 6.0 or higher, should I change minimum Tizen version to 6.0?

Reply:
Which internal API requires Tizen 5.5 for example?

https://docs.tizen.org/application/native/api/wearable/latest/group__WEBVIEW.html

> should I change minimum Tizen version to 6.0?

Not really. You should use the lowest possible version (maybe 4.0).

Also the additional information seems redundant.

```suggestion
| [**webview_flutter_tizen**](packages/webview_flutter) | 5.5 | ❌ | ❌ | ✔️ | ❌ |
```

Reply:
> Which internal API requires Tizen 5.5 for example?

`ewk_view_ime_window_set`[1424](https://review.tizen.org/gerrit/gitweb?p=platform/framework/web/chromium-efl.git;a=blob;f=tizen_src/ewk/efl_integration/public/ewk_view_internal.h;h=2bbf2a510e8f01cd43b72cb239cd9d3832cd4e84;hb=refs/heads/tizen_5.5#l1424)  * @since_tizen 5.5
`ewk_view_offscreen_rendering_enabled_set`[1414](https://review.tizen.org/gerrit/gitweb?p=platform/framework/web/chromium-efl.git;a=blob;f=tizen_src/ewk/efl_integration/public/ewk_view_internal.h;h=2bbf2a510e8f01cd43b72cb239cd9d3832cd4e84;hb=refs/heads/tizen_5.5#l1414)  * @since_tizen 5.0